.travis.yml

```diff
@@ -85,8 +85,8 @@ install:
 # any command which exits with a non-zero exit code causes the build to fail.
 script:
   # test that source-distributions can be generated
-  - (cd "." && cabal sdist)
-  - mv "."/dist/waargonaut-*.tar.gz ${DISTDIR}/
+  - cabal new-sdist all
```

Comment:
Will this cause travis to fail if we can't package up the benchmarks as a "releasable" package ? As they are listed in the cabal.project file. 

I can't think of a reason _why_ the benchmarks wouldn't be able to be packaged up, but it seems like a silly thing to fail a build for? 

Am I overthinking this?

Reply:
In this case no, but I've just realised the waarg benchmarks aren't built at all by travis.
I think we want them built, not run, and not checked for releasableness. Does that sound right.

Reply:
Yes, that would make sense.